README.md

```diff
@@ -1,6 +1,12 @@
 # ndx-nirs Extension for NWB
 
-This is an NWB extension for storing and sharing near-infrared spectroscopy (NIRS) data. 
+This is an [NWB](https://www.nwb.org/) extension for storing and sharing near-infrared spectroscopy (NIRS) data.
+
+If you're new to NWB: "Neurodata Without Borders (NWB) is a data standard for neurophysiology, providing neuroscientists with a common standard to share, archive, use, and build common analysis tools for neurophysiology data." ([source](https://www.nwb.org/nwb-neurophysiology/))
+
+This extension defines the data specification for NIRS data in addition to providing a python API for reading and writing .nwb files containing data that follows this specification. The python package can be used with [pyNWB](https://github.com/NeurodataWithoutBorders/pynwb).
+
+This extension along with others can be found in the [Neurodata Extensions Catalog](https://nwb-extensions.github.io/).
```

Comment:
This might not be the right place for this. But it would be good to say somewhere that this extension is accepted (i.e. has been reviewed and welcomed by the standard). Otherwise, it is not clear if it is in preparation or just a proposal. 

```suggestion
This extension has been officially accepted into the NWB specification and can be found in the [Neurodata Extensions Catalog](https://nwb-extensions.github.io/) along with other accepted extensions.
```

Reply:
great suggestion, thanks!

Reply:
I slightly modified the text because it has been accepted to the extension catalog but is not technically part of the NWB specification.